addon/mixins/focusing.js

```diff
@@ -1,21 +1,28 @@
-import Ember from "ember";
-import getOwner from 'ember-getowner-polyfill';
+import Ember from 'ember';
 
-const { get } = Ember;
+/* globals getOwner */
+// One of these two will be uncommented at build time:
```

Comment:
Approach from my most-recent review makes this nicer. Rather than trying to make it turnkey it does the right thing for most scenarios and tells people how to configure it for the other scenarios.